src/Provider/Metadata.php

```diff
@@ -52,7 +52,7 @@ public function __construct(
+        $this->image = $image;
```

Comment:
Why did you change this?

Reply:
That will happen when its a fallback string:
<img width="1297" alt="bildschirmfoto 2019-01-20 um 18 43 26" src="https://user-images.githubusercontent.com/10114981/51443198-7e167400-1ce5-11e9-83f3-72dbf8464186.png">

Reply:
cc @jo66 please help

Reply:
We need to allow null here, otherwise in the template `select_provider.html.twig:32`
```twig
{% if not provider.providerMetadata.image %}
    <i class="{{ provider.providerMetadata.option('class') }}"></i>
{% else %}
    <img src="{{ asset(provider.providerMetadata.image) }}" style="max-height: 20px; max-width: 100px;"/><br />
{% endif %}

```

Reply:
okay shall i change it?

Reply:
I guess it's better to keep image not null, so yes.